app/Http/Requests/StoreAssetRequest.php

```diff
@@ -20,7 +20,20 @@ public function authorize(): bool
 
     public function prepareForValidation(): void
     {
-        //
+        // this needed to be added in because the observer fails if `assigned_to` is null :shrug:
```

Comment:
Why tho?

Reply:
@snipe i removed that bit and everything seems to be working fine and checkout tests are still passing. not totally sure what it was about, it may have been failing before i finished getting the rules array set up and i put it in as a stop gap.